### ISSUE TITLE
Sending GET request to /api/v1/getLog endpoint resulted in an error: {}

### ISSUE THREAD

**Issue by kine** [vjekob/al-objid#6]
Looking today into the log console and see:

```
[2021-09-13T07:57:24.062Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T07:57:24.073Z] Executing Sending GET request to /api/v1/getLog endpoint took 351.1608999967575 milliseconds
[2021-09-13T07:57:24.073Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T07:57:39.015Z] Executing Sending GET request to /api/v1/getLog endpoint took 119.86720000207424 milliseconds
[2021-09-13T07:57:39.015Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T07:57:39.016Z] Executing Sending GET request to /api/v1/getLog endpoint took 241.2812000066042 milliseconds
[2021-09-13T07:57:39.016Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T07:57:54.029Z] Executing Sending GET request to /api/v1/getLog endpoint took 131.96009999513626 milliseconds
[2021-09-13T07:57:54.029Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T07:57:54.033Z] Executing Sending GET request to /api/v1/getLog endpoint took 267.3488999903202 milliseconds
[2021-09-13T07:57:54.033Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
```

**Comment by vjekob** [comment]
Are you by any chance using it with anything of this:
- old version of the extension (<1.0.4)
- authorized your app with version <1.0.4

**Comment by kine** [comment]
version is 1.0.4
Authorized today morning at 9:52 and I think with the new version...

Tried again with another multiroot workspace, no authorization same problem

```
[2021-09-13T09:15:54.338Z] Executing Sending GET request to /api/v1/getLog endpoint took 3196.297399997711 milliseconds
[2021-09-13T09:15:54.338Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
[2021-09-13T09:15:54.367Z] Executing Sending GET request to /api/v1/getLog endpoint took 6422.053399994969 milliseconds
[2021-09-13T09:15:54.367Z] Sending GET request to /api/v1/getLog endpoint resulted in an error: {}
```

**Comment by kine** [comment]
Even restart of the VSCode didn't help.

**Comment by kine** [comment]
Rest is working (ID assignment etc.)

**Comment by vjekob** [comment]
Hmmm... this is fishy. But thanks for testing it! I'll take a look this evening.

**Comment by kine** [comment]
Could be some Azure protection if too many connections from one IP? If you imagine 8 developers, multiple VCCode windows pooling from one IP.

**Comment by vjekob** [comment]
No, this isn't it. If it were that, it would be happening on every type of request. It's only failing for `getLog`. It's some crap I did 😁

**Comment by vjekob** [comment]
This is happening on all repos where no object IDs have been assigned. It's a back-end issue: back end is returning empty string instead of empty JSON array if there are no log entries.

This is fixed in 1.0.5 (will be deployed later today)
